api/python/provisioner/commands/setup_provisioner.py

```diff
@@ -1868,6 +1868,13 @@ def _run(self, nodes, **kwargs):  # noqa: C901 FIXME
                 "salt-call state.apply components.misc_pkgs.ipmi"
             )
 
+        logger.info("Setting unique ClusterID to pillar file")
```

Comment:
Since, we are not sure of the factory and field flow yet, we would maintain this as an explicit command.
Comment out these lines for now.

Reply:
@ypise for now, added it to bootstrap flow as we're checking node role for `set_cluster_id` and executing. Thanks